src/test/javascript/admin.controller.test.js

```diff
@@ -113,6 +113,7 @@ describe("AdminController", function () {
 
     describe("init", () => {
         it("should call groupingsService.getAdminLists", () => {
+            let res = {};
```

Comment:
Does this do anything?

Reply:
Oh, I don't think that does anything. I think it was from when we tried to fix the failing Jasmine test. I'll remove it now.